### ISSUE TITLE
Maintenance Status?

### ISSUE THREAD

**Issue by nmcbride** [codelucas/newspaper#813]
Is this library still being maintained?

Thanks

**Comment by iwpnd** [comment]
[Guess not](https://github.com/codelucas/newspaper/graphs/contributors).

**Comment by nmcbride** [comment]
If anyone who has time wants to fork it and take it under their wing that'd be cool. If you do let me know and I can try to help out when I can.

**Comment by Bachstelze** [comment]
Which branch should we maintain?
There are a bunch of pull requests.

**Comment by mercuree** [comment]
Master branch is a good way to start

**Comment by FireBelow** [comment]
I'm interested too

**Comment by nmcbride** [comment]
Who would like to take it on?

**Comment by binfeng** [comment]
I'm interested to join as well.

**Comment by codelucas** [comment]
I am interesting in setting up a team to maintain this repo with more repo ownership permissions. I'll create a poll/mailing list later around this.

I am busy with fulltime work to spend lots of time on individual maintenance, but can help with things like the direction this repo is moving in, once in a while fixing bugs, and setting up a maintenance team

**Comment by codelucas** [comment]
if you are interested, please reach out at: lucasyangpersonal@gmail.com

**Comment by sagunsh** [comment]
I am interested too in helping maintain this project.

**Comment by johnbumgarner** [comment]
Has there been any movement in maintaining this project?

**Comment by rayrrr** [comment]
Just a thought: there is a large team of volunteer Python maintainers called https://jazzband.co -- there were about 500 of us at last count. More like an army of volunteer. This could be a good fit. I'm going to email you about this as well @codelucas as I'm a maintainer there and one of the people in charge.

**Comment by johnbumgarner** [comment]
> Just a thought: there is a large team of volunteer Python maintainers called https://jazzband.co -- there were about 500 of us at last count. More like an army of volunteers. This could be a good fit. I'm going to email you about this as well @codelucas as I'm a maintainer there and I know one of the people in charge.

Good luck.  The code base for this module hasn't been updated in 2+ years, so I'm unsure of the viability of this Python Library at the moment.  There are dozens of pull requests awaiting @codelucas approval.

**Comment by Bachstelze** [comment]
In my sight the next important step in maintaining this package is to drop the python3.4 support due to lxml: https://github.com/codelucas/newspaper/pull/768 https://github.com/codelucas/newspaper/issues/733 It is still unclear if we have to add pandas as dependency.
How does jazzband organize and coordinate the projects? I would suggest that we communicate over email groups.

**Comment by FireBelow** [comment]
I don't know the details of jazzband and that might be a good option. Or we
just fork it with multiple owners, this needs more attention

**Comment by Bachstelze** [comment]
We should stay here as long as @codelucas is willing to organize the maintenance. Moving to other owners won't solve the travis blockers e.g. the [F String Invalid Syntax in Python 3.5](https://stackoverflow.com/questions/55182209/f-string-invalid-syntax-in-python-3-5). I see 3 options for this particular problem: Drop 3.5, use `fromstr` or `future-fstrings`.

Which permissions do we need to rerun the travis checks afterwards?

**Comment by FireBelow** [comment]
Has there been movement since June?

**Comment by johnbumgarner** [comment]
This code base for this module hasn't been updated in years based on the pending _pull requests._  I haven't seen any movement by  @codelucas to maintain newspaper, so does this library need to be forked and rebuilt/maintained by another team of volunteers?

**Comment by FireBelow** [comment]
My preference would be for codelucas to open it up to other owners so those
pull requests can be reviewed and approved. We need development to continue
and would prefer not to have to fork with new owners if possible.

**Comment by johnbumgarner** [comment]
I agree @FireBelow, but @codelucas seemed to have gone AWOL on this project.  He doesn't respond to emails about this open source project, triage issues or handle pull requests.  This looks to be a systemic problem based on the timestamps within this repository.

**Comment by FireBelow** [comment]
I agree, and was just trying to give @codelucas another chance to respond
and participate. Should we pick a time to fork if no response is received
by then (weeks)?

**Comment by lordrisborik** [comment]
I would like to join improvement team of newsletter3k. 
Guys let's start as soon as possible: 
Barren Wallet was asked: when were you start to buy stocks? he said When I was 6 years old. 
Do you have any regret? Yes. What was that? I should have started when I was 4-year old.

**Comment by johnbumgarner** [comment]
@codelucas  

You wrote this in June 2020:

---------------------------
> I am interesting in setting up a team to maintain this repo with more repo ownership permissions. I'll create a poll/mailing list later around this.
> 
> I am busy with fulltime work to spend lots of time on individual maintenance, but can help with things like the direction this repo is moving in, once in a while fixing bugs, and setting up a maintenance team
---------------------------

The code base for this Python module hasn't been updated for years.   Please let us know the status of this module. 

1. Do you plan on clearing the backlog of Pull Requests?  If not, how can other resolves these Pull Requests? 
2. Do you plan on maintaining this module? If not, how can others take over the controlled maintenance of this module?

And why aren't you answering emails related to the the status of this module?
